ports/stm32/stm32_nema.c

```diff
@@ -68,12 +68,16 @@ int32_t nema_sys_init(void) {
     #else
     ring_buf.bo = nema_buffer_create(OMV_GPU_NEMA_RING_SIZE);
     #endif
+    nema_buffer_map(&ring_buf.bo);
```

Comment:
This doesn't do anything, just maps virtual to physical address, which is the same so it just returns `bo->base_virt` which is unused.